src/azip/azure.go

```diff
@@ -119,14 +120,15 @@ func getNIC(nicClient network.InterfacesClient, vm compute.VirtualMachine, group
 			if err != nil {
 				return nil, err
 			}
```

Comment:
Please update the comment up above `// more than one NIC found. Look for the primary one` to align with the new behavior below.

Reply:
Done